app/assets/javascripts/components/common/date_picker.jsx

```diff
@@ -92,7 +92,7 @@ const DatePicker = createReactClass({
    * @return {String} formatted date
    */
   getFormattedDateTime() {
-    return CourseDateUtils.formattedDateTime(this.getDate(), this.props.showTime);
+    return CourseDateUtils.formattedDateTime(this.getDate().toDate(), this.props.showTime);
```

Comment:
Why is this done here instead of within `getDate()`?

Reply:
The rest of the file still uses moment js so `getDate()` still returns a moment date object.